contracts/lib/LibBalances.sol

```diff
@@ -155,6 +153,20 @@ library Balances {
         return newPosition;
     }
 
+    function getFeeRate(
+        uint256 amount,
+        uint256 executionPrice,
+        uint256 feeRate
+    ) internal pure returns (int256) {
```

Comment:
I think this should be changed to getFee or calculateFee, since it uses the "rate" to calculate the absolute amount